src/Sulu/Bundle/ContactBundle/Resources/public/js/services/account-manager.js

```diff
@@ -415,6 +306,96 @@ define([
 
             return promise;
         },
+
+        /**
+         * Removes medias from an account
+         * @param mediaIds Array of medias to delete
+         * @param accountId The account to delete the medias from
+         * @private
+         */
+        removeDocuments: function(accountId, mediaIds) {
+            if (!$.isArray(mediaIds)) {
+                mediaIds = [mediaIds];
+            }
+
+            var requests = [],
+                promise = $.Deferred();
+
+            util.each(mediaIds, function(index, id) {
+                requests.push(AccountManager.prototype.removeDocument(accountId, id, true));
+            }.bind(this));
+
+            $.when.apply(null, requests).then(function() {
+                mediator.emit('sulu.contacts.documents.removed', accountId, mediaIds);
+                mediator.emit('sulu.labels.success.show', 'contact.accounts.documents-removed');
+                promise.resolve();
+            }.bind(this));
+
+            return promise;
+        },
+
+        /**
+         * Removes media from an account
+         * @param mediaId media to delete
+         * @param accountId The account to delete the medias from
+         * @param noLabel Iff true no label is displayed
+         * @private
+         */
+        removeDocument: function(accountId, mediaId, noLabel) {
```

Comment:
thes method shoul be private. only "removeDocuments" should be executable from another component